.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+cache:
+  directories:
+    - node_modules
+notifications:
+  email: false
+node_js:
+  - '7'
+  - '6'
+before_script:
+  - npm prune
```

Comment:
This command removes "extraneous" packages. If a package name is provided, then only packages matching one of the supplied names are removed.

No entendi muy bien cuan es la idea de tener este comando aca, el texto de arriba es de https://docs.npmjs.com/cli/prune

Reply:
La verdad lo tome de otra configuración y no tiene mucho sentido aquí lo voy a eliminar